### ISSUE TITLE
<ImportError: No module named wand.image>  seen on tag v2.1.0 and not seen in tag v2.0.0

### ISSUE THREAD

**Issue by chinnabha** [OP-TEE/optee_os#995]
Hi,

I am seeing this error after I moved to tag v2.1.0. I didn't face this error when working on tag v2.0.0. Can someone please help me on this?

**make PLATFORM=hikey CFG_ARM64_core=y CFG_TEE_TA_LOG_LEVEL=4 DEBUG=1 ta-targets=ta_arm64** 
  CHK     out/arm-plat-hikey/conf.mk
  UPD     out/arm-plat-hikey/conf.mk
  CHK     out/arm-plat-hikey/include/generated/conf.h
  UPD     out/arm-plat-hikey/include/generated/conf.h
  CHK     out/arm-plat-hikey/core/include/generated/asm-defines.h
  GEN     out/arm-plat-hikey/ta_arm64-lib/libutee/tui/default_bold.c
Traceback (most recent call last):
  File "scripts/render_font.py", line 29, in <module>
    from wand.image import Image
**ImportError: No module named wand.image**
mk/subdir.mk:150: recipe for target 'out/arm-plat-hikey/ta_arm64-lib/libutee/tui/default_bold.c' failed
make: *** [out/arm-plat-hikey/ta_arm64-lib/libutee/tui/default_bold.c] Error 1

**Comment by jforissier** [comment]
https://github.com/OP-TEE/optee_os/issues/929

TL;DR: `sudo apt-get install libmagickwand-dev python-wand`

**Comment by chinnabha** [comment]
Thanks. Issue resolved.

**Comment by jeyakuna** [comment]
Window 10 Python - 3.8.2  wand 0.6.2
While creating Python Executable, I am getting this error.

Traceback (most recent call last):
  File "myapp.py", line 11, in <module>
  File "e:\apps\anaconda3\envs\envimgcomparelversion\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "ClsImgCompare.py", line 17, in <module>
ModuleNotFoundError: No module named 'wand'
[25172] Failed to execute script myapp

Please help me
